src/jagadeeshMJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,47 @@
+/*Program 2: To print first n Fibonacci Numbers.
+Input n = 5
+Output = 1 1 2  3  5  8 
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55
+
+Hint :
+The next number is found by adding up the two numbers before it:
+the 2 is found by adding the two numbers before it (1+1),
+the 3 is found by adding the two numbers before it (1+2),
+the 5 is (2+3),
+and so on! */
+package jagadeeshMJun20;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
+	private int num;
+
+	void getInput(int number) {
+		num = number;
+
+		displayOutput();
+	}
+
+	private void displayOutput() {
```

Comment:
This method works good. However I think there can be a simpler way for doing this without using an array --
for (int index = 0; index < n; index++) {
			int sum = firstNum + nextNum;
			System.out.print(nextNum + " ");
			firstNum = nextNum;
			nextNum = sum;
		}